zerver/tests/test_import_export.py

```diff
@@ -1518,6 +1519,34 @@ def test_import_realm(self) -> None:
         special_characters_message = "```\n'\n```\n@**Polonius**"
         self.send_stream_message(self.example_user("iago"), "Denmark", special_characters_message)
 
+        todo_message_topic = "TODO message"
+        todo_message_sender_name = "iago"
+        todo_widget_message = "/todo Example Task List Title\n\n    task without description\ntask: with description    \n\n - task as list : also with description"
+        export_realm_todo_message_id = self.send_stream_message(
+            self.example_user(todo_message_sender_name),
+            "Denmark",
+            todo_widget_message,
+            todo_message_topic,
+        )
+        export_realm_todo_submessage = SubMessage.objects.get(
+            message_id=export_realm_todo_message_id
+        )
+
+        poll_message_topic = "Poll message"
+        poll_message_sender_name = "hamlet"
+        poll_widget_message = (
+            "/poll What is your favorite color?\n\nRed\nGreen  \n\n   Blue\n - Yellow"
+        )
+        export_realm_poll_message_id = self.send_stream_message(
+            self.example_user(poll_message_sender_name),
+            "Denmark",
+            poll_widget_message,
+            poll_message_topic,
+        )
+        export_realm_poll_submessage = SubMessage.objects.get(
```

Comment:
can we name this variable and `export_realm_todo_submessage` without the `export_` prefix? that's more consistent with the naming convention in this test I think. it's the post-import objects that just get the `imported_....` naming.